packages/compass-indexes/src/components/indexes-table/indexes-table.tsx

```diff
@@ -100,53 +116,75 @@ export const IndexesTable: React.FunctionComponent<IndexesTableProps> = ({
     return _columns;
   }, [canDeleteIndex, onSortTable]);
 
+  useEffect(() => {
+    const container =
+      tableRef.current?.getElementsByTagName('table')[0]?.parentElement;
+    if (container) {
+      container.style.height = containerHeight;
```

Comment:
Doesn't seem like it's working when shell is expanded. Is this expected? 

![image](https://user-images.githubusercontent.com/5036933/187631854-4708e847-efb0-4a34-8beb-948aac259966.png)